src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__date-1 calendar_start-day_tue"></div>
```

Comment:
All modifiers for calendar block should be written near calendar.
And make sure to follow BEM naming convention for complex modifiers: block-name--modifier-name--modifier-value (checklist)
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--tue">
```